docs/README.md

```diff
@@ -1,27 +1,21 @@
 # Taurus Data Model Documentation
 
-This the next major data model developed by Citrine, codename `taurus`.
+This is the next major data model developed by Citrine, codename `taurus`.
 
-The format links together materials, the processes that produced them,
-and the measurements that characterize them.
-This facilitates the backwards traversal from a measurement to the material on which it was performed
-to the process by which it was produced to the materials which were used in that process and so on and so forth.
-This generalizes and matures the PIF's `preparation` and `subSystems` objects.
+The model links together materials, the processes that produced them, and the measurements that characterize them.
+This facilitates the backwards traversal from a measurement to the material on which it was performed to the process by which it was produced to the materials which were used in that process. 
+It generalizes and matures `preparation` and `subSystems` objects within the Physical Information File (PIF).
 
-Additionally, this format makes a first-class distinction between *intent* and *realization*, 
-captured by `Spec` and `Run` objects, respectively.
-A single intent `Spec` can be realized into multiple `Run` objects.
-This generalized and matures the `ideal` concept from the PIF's `Composition` and `Quantity` objects.
+Additionally, the model makes a first-class distinction between *intent* and *realization*, captured by `Spec` and `Run` objects, respectively. 
+A single intent `Spec` can be realized into multiple `Run` objects. This generalizes and matures the `ideal` concept from the PIF's `Composition` and `Quantity` objects.
```

Comment:
```suggestion
A single intent `Spec` can be realized into multiple `Run` objects.
This generalizes and matures the `ideal` concept from the PIF's `Composition` and `Quantity` objects.
```
maintain (no more than) one sentence per line

Reply:
Great! To keep it manageable, I'll remove the DO NOT MERGE and issue new PR's as I go through all of this documentation in the next week or so.